scripts/migrateCustomerDeliveryChannels.sql

```diff
@@ -0,0 +1,80 @@
+FROM "DeliveryChannelPolicies" as table_2, "Customers" AS customers
+WHERE customers."Id" <> 1 AND table_2."System" = false;
+
+-- create default channels
+
+BEGIN TRANSACTION;
+
+-- this is inserting default delivery channels into the table based on what customer 1 (the admin customer) has
+-- table_1 and table_2 are the same table
+INSERT INTO "DefaultDeliveryChannels" AS table_1 ("Id", "Customer", "Space", "MediaType", "DeliveryChannelPolicyId")
+SELECT gen_random_uuid(),customers."Id", 0, table_2."MediaType", table_2."DeliveryChannelPolicyId"
+FROM "DefaultDeliveryChannels" as table_2, "Customers" AS customers
+WHERE table_2."Space" = 0 AND table_2."Customer" = 1 AND customers."Id" <> 1;
+
+-- The next 3 queries are updating specific DDC to use the created System:false policies that were inserted
+-- when creating policies as the insert query above this, creates DDC linking JUST to policies owned by customer 1
+
+-- if you had customer 2, this would update the DDC entry for default-audio to use the DeliveryChannelPolicy created
+-- above, instead of the policy used by customer 1
+UPDATE "DefaultDeliveryChannels"
+SET "DeliveryChannelPolicyId" = joined_table."Id"
+FROM (SELECT "DeliveryChannelPolicies"."Id", DDC."Customer" FROM "DeliveryChannelPolicies"
+    JOIN public."DefaultDeliveryChannels" DDC on "DeliveryChannelPolicies"."Id" = DDC."DeliveryChannelPolicyId"
+         WHERE "Name" = 'default-audio' AND "MediaType" = 'audio/*') as joined_table
+WHERE "DefaultDeliveryChannels"."Customer" = joined_table."Customer" AND "DefaultDeliveryChannels"."Customer" <> 1
+  AND "MediaType" = 'audio/*';
+
+UPDATE "DefaultDeliveryChannels"
+SET "DeliveryChannelPolicyId" = joined_table."Id"
+FROM (SELECT "DeliveryChannelPolicies"."Id", DDC."Customer" FROM "DeliveryChannelPolicies"
+    JOIN public."DefaultDeliveryChannels" DDC on "DeliveryChannelPolicies"."Id" = DDC."DeliveryChannelPolicyId"
+         WHERE "Name" = 'default-video' AND "MediaType" = 'video/*') as joined_table
+WHERE "DefaultDeliveryChannels"."Customer" = joined_table."Customer" AND "DefaultDeliveryChannels"."Customer" <> 1
+  AND "MediaType" = 'video/*';
+
+UPDATE "DefaultDeliveryChannels"
+SET "DeliveryChannelPolicyId" = joined_table."Id"
+FROM (SELECT "DeliveryChannelPolicies"."Id", DDC."Customer" FROM "DeliveryChannelPolicies"
+    JOIN public."DefaultDeliveryChannels" DDC on "DeliveryChannelPolicies"."Customer" = DDC."Customer"
+         WHERE "Name" = 'default' AND "MediaType" = 'image/*' AND "Channel" = 'thumbs' AND "DeliveryChannelPolicyId" = 3) -- is this correct? - will always be 3 as it's set on a migration, but could be made more flexible with a SELECT
+    as joined_table
+WHERE "DefaultDeliveryChannels"."Customer" = joined_table."Customer" AND "DefaultDeliveryChannels"."Customer" <> 1
+  AND "MediaType" = 'image/*' AND "DeliveryChannelPolicyId" = 3;
+
+COMMIT;
+
+
+SELECT * FROM "DeliveryChannelPolicies"

```

Comment:
no, they aren't required, but I thought it might be useful to have some of the queries I used to check what was happening in the db - would likely be better to go on the verification script